### PR TITLE
Les 450 create connecting between FE and BE

### PR DESCRIPTION
**Pull request by Kurilo-Pavel**

### CODE REVIEW

**Review by Evgeniy121212 — APPROVED** [review]
approving ahead

**Review by valer-sky — APPROVED** [review]

File: src/pages/LoginPage.tsx [hunk 1]
```diff
@@ -52,6 +50,9 @@ const LoginPage = () => {
             errors.email =
               intl.formatMessage({id: "app.firstRegistrationForm.invalidationRules"});
           }
+          if (!isEmail && values.email.length > 0) {
```
Comment (Terapool) [inline src/pages/LoginPage.tsx:53]: if (!isEmail && values.email.length > 0) =>

if (!isEmail && values.email.length)

File: src/pages/ResetPasswordPage.tsx [hunk 2]
```diff
@@ -45,46 +37,34 @@ const ResetPasswordPage = () => {
             errors.email =
               intl.formatMessage({id: "app.firstRegistrationForm.invalidationRules"});
           }
+          if (!isEmail && isEmail!=="") {
```
Comment (Terapool) [inline src/pages/ResetPasswordPage.tsx:40]: Please, spaces over ==

File: src/translations/ru.js [hunk 3]
```diff
@@ -79,7 +76,7 @@ export const messages_ru = {
   "app.Add new lesson": "Добавить новый урок",
   "app.don'tAccount": "У вас нет аккаунта?",
   "app.code.name": "Код",
-  "app.code.errorLength": `Коде должен содержать ${CODE.maxLength} символов. `,
+  "app.code.errorLength": "Коде должен содержать {maxSymbol} символов. ",
```
Comment (Terapool) [inline src/translations/ru.js:79]: Коде ?

File: src/translations/ru.js [hunk 4]
```diff
@@ -90,16 +87,14 @@ export const messages_ru = {
   "app.resetPasswordPage.text":
     "Мы отправили ссылку для восстановления доступа к вашему аккаунту на адрес ",
   "app.setNewPasswordPage.title": "Войти как пользователь",
-  "app.YourselfPage.errorIncorrectName": "Неверное имя пользователя",
-  "app.YourselfPage.errorFieldEmpty": "Поле не должно быть пустым",
-  "app.YourselfPage.errorSmalName": `Имя пользователя должно быть длинее ${USERNAME.minLength} символов`,
-  "app.YourselfPage.errorBigName": `Имя пользователя должно быть короче ${USERNAME.maxLength} символов`,
+  "app.YourselfPage.errorIncorrectName":"Неверное имя пользователя",
```
Comment (Terapool) [inline src/translations/ru.js:90]: Please, spaces afer ":"

**Review by Terapool — CHANGES_REQUESTED** [review]

**Review by Terapool — APPROVED** [review]

**Pull request merged**